### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJEclipse_Deploy to Azure_Screen Reader: JAWS is not reading the first list items present in "Azure" submenu item

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#3379]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier!

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS1.3.1; #Screen Reader; #C+E-H; #Java tooling for IntelliJ/Eclipse; #JAWS 

**NOTE: Same is the case with all the sub menu items in the application**

**Environment Details:**
Download and Install IntelliJ IDEA from https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-create-hello-world-web-app?view=azure-java-stable
IntelliJ IDEA 2019.1.3
Build #IU-191.7479.19

**OS Details: ​**
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS 1.3.1

**Repro Steps:**
1) Start JAWS and open IntelliJ IDEA application
2) Press 'Ctrl+Alt+S' to open application settings/preferences popup
3) Tab till 'Plugins' tab and press enter
4) Search for 'Azure Toolkit for IntelliJ in the Marketplace' and press enter.
5) Tab till 'Install' button and press enter
6) Restart IntelliJ IDEA application, and tab till Tools-->Azure-->Azure Sign in
7) Sign in with any Azure subscription
8) Tab till File menu and create new maven-archetype-webapp project
9) Under Project Explorer view, open and edit the file src/main/webapp/index.jsp
10) Under Project Explorer view, tab till the created project and open the context menu
11) Tab till "Azure" and press right arrow to expand it. The focus will land on "Deploy to Azure"
12) Check if JAWS is reading the list items 

**Actual Result:**
JAWS is not reading the list item "Deploy to Azure"

**Expected Result:**
JAWS should read the list item when it is focused

**User Impact:**
Screen Reader users will not know whether the focus is on list item or not

**MAS Reference:**
[MAS 1.3.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4%7D&cid=c1d83660-4891-4456-916d-4e23c4c6cd8a)

[Bug_3379_IntelliJ_Azure_SubMenu_JAWS.pptx](https://github.com/microsoft/azure-tools-for-java/files/3461046/Bug_3379_IntelliJ_Azure_SubMenu_JAWS.pptx)
[Bug_3379_IntelliJ_New Project_SubMenu_JAWS.pptx](https://github.com/microsoft/azure-tools-for-java/files/3461047/Bug_3379_IntelliJ_New.Project_SubMenu_JAWS.pptx)

**Comment by andxu** [comment]
@soumyatippapuram , I have tested the Intellj without installing our extensions, create a new maven-archetype-webapp project, and open the context menu while editing `src/main/webapp/index.jsp`, tab to "Local History"  and press right arrow to expand it. The focus will land on "Show History", the first time, "Show History" is not read. press left arrow and then press right right to expand it again, the second time "Show History" is not read, our menu of "Deploy to Azure" have the same result, the menu we contribute to IntelliJ is through Intellj-exposed API, we have no control to the ui element of the menu, so I think we can open an issue [here](https://youtrack.jetbrains.com/issues) at idea.

![image](https://user-images.githubusercontent.com/17075863/64509569-9984cc80-d312-11e9-86f0-3e858f3e35ef.png)

**Comment by andxu** [comment]
Importance of experience: Little, the menu will be read by JAWS if user uses down and up arrow button.
Quality of experience: Little, use a down arrow button and then a up arrow button will read the menu.
Alternate solutions: use a down arrow button and then a up arrow button

**Comment by andxu** [comment]
Importance of experience: Little, user can use down arrow and up arrow key to find which control is under focus.
Quality of experience: Little, the color in high contrast theme has the indication of current focus.
Alternate solutions: No alternate technologies or options can be leveraged.

**Comment by Flanker32** [comment]
@soumyatippapuram This is an external issue of IntelliJ and has been linked to AAT, please help us review it, thanks!

**Comment by soumyatippapuram** [comment]
@andxu created a new issue https://youtrack.jetbrains.com/issue/IDEA-222353

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and this is non blocking scenario

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749773
